src/extlibs/hrvo/hrvo_agent.h

```diff
@@ -143,8 +143,22 @@ class HRVOAgent : public Agent
 
         // The number of the second velocity obstacle.
         int velocityObstacle2_;
```

Comment:
If you're familiar with the usage of this class, can we update the comments in this class to be more descriptive about what a Candidate is, and what the two integers represent (VO index!?)?
`position_` should maybe be updated to `velocity`, as I believe that the candidate point actually represent a candidate velocity.

Reply:
updated